libs/components/packages/package.json

```diff
@@ -84,9 +84,9 @@
       "@skyux/tiles": "0.0.0-PLACEHOLDER",
       "@skyux/toast": "0.0.0-PLACEHOLDER",
       "@skyux/validation": "0.0.0-PLACEHOLDER",
-      "ag-grid-angular": "^31.1.1",
-      "ag-grid-community": "^31.1.1",
-      "ag-grid-enterprise": "^31.1.1",
+      "ag-grid-angular": "~31.2.0",
+      "ag-grid-community": "~31.2.0",
+      "ag-grid-enterprise": "~31.2.0",
```

Comment:
Switching the range so we can review minor versions before they are pulled in.

Reply:
Good call.